packages/alias/README.md

```diff
@@ -122,6 +122,50 @@ To replace extensions with another, a pattern like the following might be used:
 
 This would replace the file extension for all imports ending with `.js` to `.wasm`.
 
+## Custom resolver instead of built-in algorithm
+
+In some situations you would like to keep preferred resolving method together with aliasing.
+It could be done with `customResolver` option.
+
+Example:
+```javascript
+// rollup.config.js
+import alias from "@rollup/plugin-alias";
+import resolve from "rollup-plugin-node-resolve";
+
+const customResolver = resolve({
+  extensions: [".mjs", ".js", ".jsx", ".json", ".sass", ".scss"]
+});
+const projectRootDir = path.resolve(__dirname);
+
+export default {
+  // ...
+  plugins: [
+    alias(
+      {
+        entries: [
+          {
+            find: "src",
+            replacement: path.resolve(projectRootDir, "src")
+            // OR place `customResolver` here. See explanation below.
+          }
+        ]
+      },
+      customResolver
```

Comment:
this needs to be added to the ##Options and documented in the same fashion as the other options. you can then link to an expanded section like we did for the regular expressions. This section of the readme needs some error correction as well. Please give it another read through.

Reply:
@shellscape, please find changes in latest commit.  Thanks for pointing on mistakes.